task/cargo/0.1/tests/run.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: cargo-test-pipeline
+spec:
+  workspaces:
+    - name: source
+      description: The workspace that contains your cargo.toml and rust code.
+  params:
+    - name: GIT_URI
+      type: string
+      description: The git url that contains your source code.
+    - name: COMPILE_FLAGS
+      type: array
+      default: []
+  tasks:
+    - name: git-clone
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: source
+      params:
+        - name: url
+          value: '$(params.GIT_URI)'
+    - name: cargo-build
+      taskRef:
+        name: cargo
+      params:
+        - name: COMPILE_FLAGS
+          value: ['$(params.COMPILE_FLAGS[*])']
+      runAfter:
+        - git-clone
+      workspaces:
+        - name: source
+          workspace: source
```

Comment:
is it possible to add a task that runs after cargo-build that can verify that the cargo build worked? (np if not)

Reply:
Thanks for taking a look! 

Good idea, that's a helpfull addition indeed. I've added a new task to the test pipeline that checks for the expected compiled binaries as a simple smoke test for now.